nodejs-server/routes/photos.js

```diff
@@ -0,0 +1,78 @@
+var counter  = require('counter');
+var express = require('express');
+var fs = require('fs');
+var multer  = require('multer');
+var path = require('path');
+var router = express.Router();
+
+var MQTT = require('../src/MQTT');
```

Comment:
are we using MQTT for acknowledging only (e.g., confirming the storing of image on the pi)?

Reply:
At the moment, yes, since other coms between the iPhone and pi are more of a request/response behavior (e.g. get list of image thumbnails), I believe http is more a more suited coms for our scenarios

As for using MQTT between the server and the pi app, maybe there's a better way of doing this  ¯\_(ツ)_/¯ I haven't tried just listening for file changes but ideally I want to only restart the slideshow after the complete upload of images have finished vs at each file save (which is what's currently in this PR)